### ISSUE TITLE
Error while creating CREFs

### ISSUE THREAD

**Issue by bobvanluijt** [semi-technologies/weaviate#935]
URL: `http://[IP]/weaviate/v1/things`

body: 
```
{
    "id": "5281fb0d-e17a-5a84-a554-f7c841e1799b",
    "class": "City",
    "schema": {
        "name": "Hamburg",
        "inCountry": {
            "$cref": "weaviate://localhost/things/f5283f50-60c2-5016-800c-ea97c4eee6b6"
        }
    }
}
```

Error:

```
{"error":[{"message":"could not store thing: server error: {\"message\":\"Error encountered evaluating script: g.addV(\\\"class_2\\\").as(\\\"classToBeAdded\\\").property(\\\"kind\\\", \\\"thing \\\").property(\\\"uuid\\\", \\\"5281fb0d-e17a-5a84-a554-f7c841e1799b\\\").property(\\\"classId\\\", \\\"class_2\\\ ").property(\\\"creationTimeUnix\\\", (long) 0).property(\\\"lastUpdateTimeUnix\\\", (long) 0).property(\\\"prop_13 \\\", \\\"Hamburg\\\").addE(\\\"prop_14\\\").from(\\\"classToBeAdded\\\").to(g.V().has(\\\"uuid\\\", \\\"f5283f50-6 0c2-5016-800c-ea97c4eee6b6\\\")).property(\\\"refId\\\", \\\"prop_14\\\").property(\\\"\\\\$cref\\\", \\\"f5283f50- 60c2-5016-800c-ea97c4eee6b6\\\").property(\\\"refType\\\", \\\"thing\\\").property(\\\"locationUrl\\\", \\\"localho st\\\")\",\"Exception-Class\":\"java.lang.NullPointerException\",\"exceptions\":[\"java.lang.NullPointerException\" ],\"stackTrace\":\"java.lang.NullPointerException\\n\\tat org.janusgraph.graphdb.transaction.StandardJanusGraphTx.v erifyAttribute(StandardJanusGraphTx.java:562)\\n\\tat org.janusgraph.graphdb.relations.AbstractTypedRelation.proper ty(AbstractTypedRelation.java:125)\\n\\tat org.apache.tinkerpop.gremlin.structure.util.ElementHelper.attachProperti es(ElementHelper.java:285)\\n\\tat org.janusgraph.graphdb.util.ElementHelper.attachProperties(ElementHelper.java:63 )\\n\\tat org.janusgraph.graphdb.vertices.AbstractVertex.addEdge(AbstractVertex.java:168)\\n\\tat org.janusgraph.gr aphdb.vertices.AbstractVertex.addEdge(AbstractVertex.java:37)\\n\\tat org.apache.tinkerpop.gremlin.process.traversa l.step.map.AddEdgeStep.map(AddEdgeStep.java:111)\\n\\tat org.apache.tinkerpop.gremlin.process.traversal.step.map.Ad dEdgeStep.map(AddEdgeStep.java:50)\\n\\tat org.apache.tinkerpop.gremlin.process.traversal.step.map.MapStep.processN extStart(MapStep.java:37)\\n\\tat org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(Abs tractStep.java:143)\\n\\tat org.apache.tinkerpop.gremlin.process.traversal.util.DefaultTraversal.hasNext(DefaultTra versal.java:192)\\n\\tat org.apache.tinkerpop.gremlin.util.iterator.IteratorUtils.fill(IteratorUtils.java:62)\\n\\t at org.apache.tinkerpop.gremlin.util.iterator.IteratorUtils.list(IteratorUtils.java:85)\\n\\tat org.apache.tinkerpo p.gremlin.util.iterator.IteratorUtils.asList(IteratorUtils.java:382)\\n\\tat org.apache.tinkerpop.gremlin.server.ha ndler.HttpGremlinEndpointHandler.lambda$channelRead$1(HttpGremlinEndpointHandler.java:247)\\n\\tat org.apache.tinke rpop.gremlin.util.function.FunctionUtils.lambda$wrapFunction$0(FunctionUtils.java:36)\\n\\tat org.apache.tinkerpop. gremlin.groovy.engine.GremlinExecutor.lambda$eval$0(GremlinExecutor.java:269)\\n\\tat java.util.concurrent.FutureTa sk.run(FutureTask.java:266)\\n\\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)\\n\\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\\n\\tat java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1149)\\n\\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:6 24)\\n\\tat java.lang.Thread.run(Thread.java:748)\\n\"}"}]}
```

Blocker because it is needed for a use case.

**Comment by etiennedi** [comment]
Since this error only occurs with Janusgraph 0.3.1 in combination with ES 6.x.x, I'm suggesting we downgrade all setups to use the "old" setup using ES5 with Janus to buy us some time to investigate what is really going on here.

**Comment by bobvanluijt** [comment]
Check

**Comment by etiennedi** [comment]
I've removed the Blocker label, as downgrading the Janus/ES stack seemed to indeed have fixed the problem.

Additionally I'd propose the following:
1. Make a new Helm release using ES 6.4.2 as this works much better than 6.8.1 with Janus (<-- this is very little effort)
1. Whenever we set up a production or customer UC deployment, we manually try to orchestrate the order of rollouts (first ES/C*, then Janus, Weaviate last), as this seems to yield the best results

The second point is of course something that could be automated, however given the fact that a) the amount of production-like use cases is still easily manageable and b) we are phasing out the Janus/ES stack, I don't think we can justify spending time on a proper automation.
